stories/stepforms/filter-stepform.js

```diff
@@ -0,0 +1,35 @@
+import { FilterStepForm, registerModule } from '../../dist/storybook/components';
+import { storiesOf } from '@storybook/vue';
+import Vuex from "vuex";
+
+const stories = storiesOf('Step forms/Filter step form', module);
+
+stories.add('FilterStepForm', () => ({
+  template: `
+    <FilterStepForm />
+  `,
+
+  components: {
+    FilterStepForm
+  },
+
+  store: new Vuex.Store(),
+
+  created() {
+    registerModule(this.$store, {
+      dataset: {
+        headers: [],
+        data: [],
+      },
+      availableVariables: [{
+        identifier: 'foo',
+        value: 'bar',
+        label: 'Foo',
+      }],
+      variableDelimiters: {
+        start: '{{',
+        end: '}}'
+      }
+    })
+  },
+}));
```

Comment:
I have added a very light story in the storybook for the FilterStepForm, so it's easy to test and see in FeaturePeek